akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -51,13 +51,23 @@ message Payload {
   optional bytes messageManifest = 4;
 }
 
+message WatcherHeartbeatResponse {
+  required uint64 uid = 1;
+}
+
 message Throwable {
   required string className = 1;
   optional string message = 2;
   optional Payload cause = 3;
   repeated StackTraceElement stackTrace = 4;
 }
 
+message ActorInitializationException {
+  optional ActorRef actor = 1;
```

Comment:
Hm. They don't really matter for protobuf though. I tried to keep the original field order of the corresponding system messages/exceptions where it was possible. Not fully consistent though since system messages reuse some common fields across types...